kubernetes/gke-utility/argocd/clusters.yaml

```diff
@@ -51,3 +51,28 @@ stringData:
       }
     }
 ---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: ibm-ppc64le
+spec:
+  target:
+    template:
+      engineVersion: v2
+      data:
+        name: ibm-ppc64le
+        server: https://73725434-jp-osa.lb.appdomain.cloud:6443
```

Comment:
I guess the cluster creation was handled differently since I don't see a HCL reflecting it ? So what ever happens to this cluster is not on SIG K8S Infra ?

Reply:
There isn't an IBM managed cluster with this architecture, so it's a cluster built from VMs. I did ask for the TF code for the infra to be merged into this repository.